src/Console/ConsoleIo.php

```diff
@@ -205,6 +205,13 @@ public function overwrite($message, $newlines = 1, $size = null)
         if ($newlines) {
             $this->out($this->nl($newlines), 0);
         }
+
+        // Store length of content + fill so if the new content
+        // is shorter than the old content the next overwrite
+        // will work.
+        if ($fill > 0) {
```

Comment:
Couldn't you just add the value of ``$fill``? If it's zero, it would still add up fine 😼